lib/octokit/client/milestones.rb

```diff
@@ -44,7 +44,7 @@ def milestone(repository, number, options = {})
       # @param repository [String, Repository, Hash] A GitHub repository.
       # @param title [String] A unique title.
       # @param options [Hash] A customizable set of options.
-      # @option options [String] :state (open) State: <tt>open</tt> or <tt>closed</tt>.
+      # @option options [String] :state (open) State: <tt>open</tt>, <tt>closed</tt>, or <tt>all</tt>.
```

Comment:
The `all` value works for listings, but isn't a valid value for specifying state on a milestone.